### PR TITLE
Fix for 'iPad' showing as the browser on iPad.

### PR DESCRIPTION
**Pull request by lopesc**
Added Safari as the browser name (this is expected) so that it is consistent with the results for other devices.
Updated tests accordingly.

### CODE REVIEW

**Comment by lopesc** [comment]
Fixes issue where the expected browser name for some iOS devices was being returned as the name of the device instead of the browser (eg, iPad instead of Safari).  Fix now ensures that the browser name is returned for these devices - device information is available in the results in the ipad, iphone, and ipod properties.  

See #55 for more info.

**Comment by lopesc** [comment]
@ded Can you take a look?

**Comment by vzaidman** [comment]
what about line 172:
else if (/safari/i.test(ua)) {
      result = {
        name: 'Safari'
      , safari: t
      , version: versionIdentifier
      }
    }

is it relevant at all?

**Comment by lopesc** [comment]
@vzaidman Yes it is.  The fix here addresses only iOS devices (i.e. iPad, iPhone, iPod).  The line you refer to tests the browser name on non-iOS devices.

File: src/bowser.js [hunk 1]
```diff
@@ -88,7 +88,8 @@
     }
     else if (iosdevice) {
       result = {
-        name : iosdevice == 'iphone' ? 'iPhone' : iosdevice == 'ipad' ? 'iPad' : 'iPod'
+        name : 'Safari'
```
Comment (jacksonrayhamilton) [inline src/bowser.js]: Browsers other than Safari could be used on iPhone / iPad; for instance, Chrome for iOS.

**Comment by lopesc** [comment]
@ded

**Pull request closed without merging**